go/Makefile

```diff
@@ -166,6 +166,7 @@ $(gen_sum): $(gen_src)
 	  docker run \
 	    --user="$$uid" \
 	    --volume="`go env GOPATH`/pkg/mod:/go/pkg/mod" \
+		--volume="$(GOPATH)/src/github.com/ipfs-shipyard/gomobile-ipfs:/go/src/github.com/ipfs-shipyard/gomobile-ipfs" \
```

Comment:
I think that you should use the `go list` command here (as we've already done somewhere else in the makefiles)

Reply:
this was for testing purpose, since I was using local `gomobile-ipfs` repo, i will remove this change, I dont need it anymore !